src/main/java/com/minecolonies/core/items/MineColoniesArmorMaterial.java

```diff
@@ -1,92 +1,96 @@
 package com.minecolonies.core.items;
 
+import net.minecraft.Util;
+import net.minecraft.sounds.SoundEvent;
+import net.minecraft.util.LazyLoadedValue;
 import net.minecraft.world.item.ArmorItem;
+import net.minecraft.world.item.ArmorItem.Type;
 import net.minecraft.world.item.ArmorMaterial;
 import net.minecraft.world.item.crafting.Ingredient;
-import net.minecraft.sounds.SoundEvent;
 import org.jetbrains.annotations.NotNull;
 
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.function.Supplier;
+
 /**
- * Implementation for the IArmorMaterial interface so we can add custom armor types.
+ * Implementation for the ArmorMaterial interface, so we can add custom armor types.
  */
 public class MineColoniesArmorMaterial implements ArmorMaterial
```

Comment:
just side question: side you edit this class anyway how about making it true record?

Reply:
I prefer to not hide code :D

Reply:
My intention was to leave it the same implementation as the ArmorMaterials enum so it's easy to spot differences.